spec/requests/well_known/node_info_spec.rb

```diff
@@ -19,6 +19,10 @@
             include(
               rel: 'http://nodeinfo.diaspora.software/ns/schema/2.0',
               href: include('nodeinfo/2.0')
+            ),
+            include(
+              rel: 'https://www.w3.org/ns/activitystreams#Application',
+              href: end_with('/actor')
```

Comment:
I'm not entirely keen on this, but for some reason using the `instance_actor_url` call here was generating a different URL to the one the discovery serialiser would generate, and I've no idea why, and couldn't find any explanation for this across the codebase. Perhaps there's a better solution possible for this test.